MultiGoogleTranslate.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+
+"""
+Use Google Translate to translate your sentence into multiple languages. 
+Visit the following link to check available languages: 
+https://cloud.google.com/translate/docs/languages
+To add or remove languages use modifier key when trigger is activated or go to: 
+'~/.config/albert/org.albert.extension.translate/config.json'
+Add or remove elements based on the ISO-Codes that you found on the google documentation page.
+"""
+
+from albertv0 import *
+import json
+import urllib.request
+import urllib.parse
+import os
+
+__iid__ = "PythonInterface/v0.1"
+__prettyname__ = "Multi Google Translate"
+__version__ = "1.0"
+__trigger__ = "mtr"
+__author__ = "David Britt, Manuel Schneider"
+__dependencies__ = []
+
+ua = "Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/62.0.3202.62 Safari/537.36"
+urltmpl = "https://translate.googleapis.com/translate_a/single?client=gtx&sl=auto&tl=%s&dt=t&q=%s"
+urlbrowser = "https://translate.google.com/#auto/%s/%s"
+
+configuration_directory = os.path.expanduser("~/.config/albert/org.albert.extension.mtr")
```

Comment:
please use [Extension interface (0.2)](https://albertlauncher.github.io/docs/extensions/python/#extension-interface-02) to get the config path.